### ISSUE TITLE
[css-fonts-5] <meta text-scale> limits

### ISSUE THREAD

**Issue by w3cbot** [w3c/a11y-tracking#300]
**This is a tracker issue.** Only discuss things here if they are a11y group internal meta-discussions about the issue. **Contribute to the actual discussion at the following link:**

§ https://github.com/w3c/csswg-drafts/issues/13557

**Comment by AutoSponge** [comment]
separate magnification tools get messed up when the site assumes large OS font settings should be applied to the browser. I raised this as a warning but may have picked the wrong time to bring it up and it gets lost/buried.

**Comment by AutoSponge** [comment]
I still think this needs either an opt-in or opt-out both at the all sites and individual site level to remain helpful to some users without harming others. Not sure if that's going to be part of the CSS proposal or some later consideration for implementors.

**Comment by AutoSponge** [comment]
Honestly, a CSS var that has the user's preference can be used in min/max and calc to do proper, tested typography. I think this proposal goes way too far and will only create problems. Apparently, that's already been considered and was dismissed in an explainer. I don't think WCAG or ARIA is going to suggest using this. Opposite. They would say this is a failure.
